addon/-private/system/relationships/state/relationship.js

```diff
@@ -346,7 +346,7 @@ export default class Relationship {
     this.store._updateRelationshipState(this);
   }
 
-  updateLink(link) {
+  updateLink(link, initial) {
```

Comment:
https://github.com/emberjs/data/blob/master/addon/-private/system/relationships/state/create.js#L72

Furthermore, this line and the one in my PR are the _only_ places that call `Relationship#push` in the test suite.